doc-High_Availability_Guide/topics/Renaming_HA_appliances.adoc

```diff
@@ -0,0 +1,144 @@
+
+Repeat these steps on any additional standby database-only appliances.
+
+[IMPORTANT]
+====
+Your primary and standby appliances must be reachable to each other by their hostnames. If all appliances are on the same network and are resolvable, no additional updates are needed. If your appliances exist on different networks, edit the `/etc/hosts` file on each database appliance to include entries for the IP address and hostname of each other database appliance in the cluster.
+====
+
+Proceed to re-configure high availability on the primary and standby database-only appliances.
+
+=== Re-configuring High Availability on Database-Only Appliances
+
+Re-configure replication on the database-only appliances, and restart services on your cluster.
+
+==== Configuring the Primary Database-Only Appliance
+
+On the primary database-only appliance, initialize the nodes in the database cluster to re-configure the database replication:
+
+. In the appliance console menu, select *Configure Database Replication*. 
+. Select *Configure Server as Primary*.
+. Set a unique identifier number for the server and enter the database name and credentials:
+.. Select a number to uniquely identify the node in the replication cluster.
+.. Enter the cluster database name.
+.. Enter the cluster database username.
+.. Enter the cluster database password and confirm the password.
+.. Enter the new primary database-only appliance hostname or IP address.
++
+[NOTE]
+====
+The hostname or IP address must be visible to all appliances that communicate with this database, including the {product-title_short} appliances and any global region databases.
+====
++
+.. Confirm that the replication server configuration details are correct, and select `y` to apply the configuration.
+
+
+
+==== Configuring the Standby Database-Only Appliance
+
+The steps to re-configure the standby database-only appliances are similar to that of the primary database-only appliance, in that they prepare the appliance to be database-only, but as the standby.
+
+On the standby database-only appliances, configure the following:
+
+. In the appliance console menu, select *Configure Database Replication*. 
+. Select *Configure Server as Standby*.
+. Select the database disk. {product-title_short} then activates the configuration.
+. Set a unique identifier number for the standby server and enter the database name and credentials:
+.. Select a number to uniquely identify the node in the replication cluster.
+.. Enter the cluster database name.
+.. Enter the cluster database username.
+.. Enter the cluster database password.
+.. Enter the new primary database-only appliance hostname or IP address.
+.. Enter the new standby database-only appliance hostname or IP address.
++
+[NOTE]
+====
+The hostname or IP address must be visible to all appliances that communicate with this database, including the engine appliances and any global region databases.
```

Comment:
Do we want to direct users to documentation on performing this action?

Reply:
Hmm, that's a tough one  (it's actually a note that appears 5x in the HA guide) -- I'm not sure what docs we could point them to. I always saw this as just something that users needed to be aware of, but didn't need to configure themselves, though I could be totally wrong. Do you have any ideas?

Reply:
My assumption is a reference architecture, but if this isn't something easily identified in the interim, lets make a note of it and explore if we need to expand docs to address this matter.

Reply:
Sounds good. I've created this BZ to track this for a future fix: https://bugzilla.redhat.com/show_bug.cgi?id=1686709